fdm-core/src/db/schema.ts

```diff
@@ -6,12 +6,12 @@ export const fdmSchema = pgSchema('fdm-dev')
 export type fdmSchemaTypeSelect = typeof fdmSchema
 
 // Define farms table
-export const sectorEnum = fdmSchema.enum('b_sector', ['diary', 'arable', 'tree_nursery', 'bulbs'])
-
```

Comment:
good to remove. interesting to see that it has the same spelling error as BBWP long time ago.. diary rather than dairy ;-)

Reply:
Yes, I have some words that I always misspelling ;)